analyzer/base-analyzer/src/main/java/eu/fasten/analyzer/baseanalyzer/MavenCoordinate.java

```diff
@@ -276,7 +276,7 @@ public String toPomUrl(String repo) {
 
                 final var dependency = new RevisionCallGraph.Dependency(
                         "mvn",
-                        groupId + "." + artifactId,
+                        groupId + ":" + artifactId,
```

Comment:
I don't think this works on Windows. See this: https://en.wikipedia.org/wiki/Filename#In_Windows

Reply:
It won't be used as a filename. It's just for the purpose of being able to differentiate groupId and artifactId in product name and dependency name.

Reply:
Ok, let's merge then